### PR TITLE
add when it was played for spotify in recent mode

### PR DESCRIPTION
**Pull request by Milo123459**
**This pr only effects spotify in recent mode**

This displays when the music was played!

### CODE REVIEW

File: source/plugins/music/index.mjs [hunk 1]
```diff
@@ -13,7 +13,9 @@
       embed:/^\b$/,
     },
   }
-
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc.js';
+dayjs.extend(utc);
```
Comment (lowlighter) [inline source/plugins/music/index.mjs:18]: Would it be ok to import **dayjs** and configure it in [source/app/metrics/utils](https://github.com/lowlighter/metrics/blob/master/source/app/metrics/utils.mjs#L10-L12) instead? 
That way other plugins will be able to use dayjs already configured without requiring additional imports (a bit like it's done for axios, puppetter, etc) 🙂

File: source/templates/classic/partials/music.ejs [hunk 2]
```diff
@@ -20,12 +20,15 @@
...
                 <div class="track">
-                  <img src="data:image/png;base64,<%= artwork %>" width="32" height="32" alt=""/>
+                  <img src="data:image/png;base64,<%= artwork %>" width="<%= !!played_at.length ? 48 : 32 %>" height="<%= !!played_at.length ? 48 : 32 %>" alt=""/>
                   <div class="infos">
-                    <div class="name"><%= name %></div>
+                    <div class="name"><b><%= name %><b></div>
                     <div class="artist"><%= artist %></div>
+                    <% if (played_at.length) {
+                        <div class="played_at"><%= played_at %></div>
+                    <% } %>
```
Comment (lowlighter) [inline source/templates/classic/partials/music.ejs:31]: Do you think you can set this as option like `plugin_music_played_at` or `plugin_music_last_played` (you can see this [commit](https://github.com/lowlighter/metrics/commit/64e8afec0d131198aafeeb13e7ea8eb1304fdd89) on how to add a new input, basically you just need to create a new entry in metadata.yml and retrieve its value [here](https://github.com/lowlighter/metrics/blob/master/source/plugins/music/index.mjs#L39)) so users can configure it?

Sorry for small nitpicks, the rest is good to me 👍

File: source/plugins/music/index.mjs (hunk 1 above)
Comment (Milo123459) [reply source/plugins/music/index.mjs:18]: I really don't know how but I'll take a look :)

File: source/templates/classic/partials/music.ejs (hunk 2 above)
Comment (Milo123459) [reply source/templates/classic/partials/music.ejs:31]: Will do!

**Review by lowlighter — APPROVED** [review]

File: source/templates/classic/partials/music.ejs [hunk 3]
```diff
@@ -20,12 +20,15 @@
...
-              <% for (const {name = "", artist = "", artwork = ""} of plugins.music.tracks) { %>
+              <% for (const {name = "", artist = "", artwork = "", played_at = ""} of plugins.music.tracks) { %>
                 <div class="track">
-                  <img src="data:image/png;base64,<%= artwork %>" width="32" height="32" alt=""/>
+                  <img src="data:image/png;base64,<%= artwork %>" width="<%= !!played_at.length ? 48 : 32 %>" height="<%= !!played_at.length ? 48 : 32 %>" alt=""/>
                   <div class="infos">
-                    <div class="name"><%= name %></div>
+                    <div class="name"><b><%= name %><b></div>
                     <div class="artist"><%= artist %></div>
+                    <% if (played_at.length) {
```
Comment (lowlighter) [inline source/templates/classic/partials/music.ejs:29]:
```suggestion
                    <% if (played_at.length) { %>
```

Comment (lowlighter) [reply source/templates/classic/partials/music.ejs:29]: You're missing a closing EJS tag, here that's why the build are failing 🙂

**Comment by Milo123459** [comment]
That's why it was failing!

**Review by lowlighter — APPROVED** [review]

**Comment by lowlighter** [comment]
Thanks for contributing!

**Comment by Milo123459** [comment]
Wait, @lowlighter when will this be available? Like, can I do plugin_music_played_at: yes in my workflow and it'll start "working"?

**Comment by lowlighter** [comment]
Currently your addition is available on the following:
- `lowlighter@master` 
- `Milo123459@update-spotify-` (your PR branch)
- `Milo123459@master`  (if you're up-to-date)

It's not yet available on `lowlighter@latest` as it stills points to v3.3. 
I usually try to integrate several bugfixes/features before publishing a new release so it lets a bit of time for testing and ensuring that releases won't be too buggy and break everyone's workflow
You can see `@master` as a beta version and `@latest` as a stable version 🙂

So tldr; use either `lowlighter@master`  or `Milo123459@master` after pulling your changes in it to use your new option

**Comment by Milo123459** [comment]
Cool! Thanks :D

**Pull request merged**